src/pages/profile-page/personal-profile-page/personal-profile-page.tsx

```diff
@@ -39,16 +39,21 @@ const AsyncPersonalProfilePage = () => {
     fetchDecksAndRefresh();
   }, []);
 
+  if (userData === undefined) {
+    return <LoadingPage />;
```

Comment:
was this needed? does suspense not handle this for us?

Reply:
oh I see. it's fine to keep, the only time that `userData` can be `undefined` is if the auth token expires in which case we shouldn't be able to hit this page anyhow